[shopsys] add checker to check availability of redis (#815)
shopsys/shopsys: docs/upgrade/UPGRADE-unreleased.md

- *(optional)* add a new [script](https://github.com/shopsys/shopsys/pull/759/files#diff-e5f46a7c45e95214037078344ce17721) to `scripts/install.sh`
    - this script serves as a fast way to install demo instance of Shopsys Framework.
    - also this script can be used if you change the configuration of docker or app, script will apply all the changes done in these files and rebuild images.

> - upgrade redis extension version in your `DockerFile` to version `4.1.1`

Author: add PR headline

```diff
-       RUN pecl install redis-4.0.2 && \
+       RUN pecl install redis-4.1.1 && \
```

> - add a new phing target `redis-check` to your `build-dev.xml` and use it where you need to check if Redis is running

Reviewer: ```suggestion
- add a new phing target `redis-check` to your `build-dev.xml` and use it before any call to Redis like `redis-check`
```

Author: I changed the message
